### ISSUE TITLE
Review purpose of the Community Backlog

### ISSUE THREAD

**Issue by calvin-lau-sig7** [alphagov/govuk-design-system#1821]
Community backlog includes:
- [Github project with candidate components sorted into columns](https://github.com/alphagov/govuk-design-system-backlog/projects/1)
- [Backlog website page, with candidate components listed with status flags](https://design-system.service.gov.uk/community/backlog/)

## What

It's unclear what the purpose of the backlog is it meant to:
- track what the Design System team is working on? (because it does not)
- show what the Design System team has agreed to work on? (because we have not)
- show what the community is working or wants to work on?
- bring members of the community into a project?

## Why

It's difficult to decide what we need to do until we understand what we want it to achieve.

## Who needs to know about this

Product manager, Delivery manager, community manager. Possibly content designer, designer, developer

## Done when

- [ ] Review goals
- [ ] Assess how effective current backlog(s) are meeting them
- [ ] Identify things to improve
- [ ] Decide what to do next

**Comment by christopherthomasdesign** [comment]
## Review of user-facing content about the community backlog

According to the [backlog page](https://design-system.service.gov.uk/community/backlog/), the purpose of the community backlog is to:
* help the community propose, develop or contribute to new and existing patterns and components
* make sure that the patterns and components in the Design System are built upon the research and experience of teams across the whole of government

On the [backlog on Github](https://github.com/alphagov/govuk-design-system-backlog/projects/1), we say "This is a community backlog of components and patterns being developed by the government community for the GOV.UK Design System". This isn't entirely true, only a very small number are actively being worked on at any one time. But it suggests part of the backlog's purpose is for users to see what components and patterns might be upcoming or being considered.

**Comment by CharlotteDowns** [comment]
We believe the backlog is currently being used in three distinct ways as a:

- space for sharing examples and iterations
- feedback loop for user research
- way of proposing new things to the Design System

**Comment by kellylee-gds** [comment]
We think it's also being used as a comms mechanism for our team eg when there is an accessibility review, working group review, design rationale on an iteration to an existing component or pattern

**Comment by kellylee-gds** [comment]
Agreed to scope the work in Q2 to focus on improving the community backlog as a proposal space. Cards currently prioritised; #1876 #1878

Shall we close this card @CharlotteDowns ?
